functional/FunctionalTest.java

```diff
@@ -668,10 +668,41 @@ public static void listObject_test4() throws Exception {
   }
 
   /**
-   * Test: listObjects(bucketName, final String prefix, final boolean recursive, final boolean useVersion1).
+   * Test: listObjects(bucketName, final String prefix, final boolean recursive).
    */
   public static void listObject_test5() throws Exception {
     int i;
+    int totalObjsNr = 1050;
+
+    System.out.println("Test: listObjects(final String bucketName, final String prefix, final boolean recursive)");
+    String[] filenames = new String[totalObjsNr];
+    for (i = 0; i < totalObjsNr; i++) {
+      String filename = createFile(1);
+      client.putObject(bucketName, filename, filename);
+      Files.delete(Paths.get(filename));
+      filenames[i] = filename;
+    }
+
+    i = 0;
+    for (Result<?> r : client.listObjects(bucketName, "minio", true)) {
+      ignore(i++, r.get());
+    }
+
+    // Check the number of uploaded objects
+    if (i != totalObjsNr) {
+      throw new Exception("[FAILED] Test: listObject_test5(), number of items found: " + i);
```

Comment:
You could indicate `expected` in the error message `"[FAILED] Test: listObject_test5(), expected object count:  " + objCount + ", got: " + i`